packages/fast-form-generator-react/src/form/form-item.text-align.tsx

```diff
@@ -36,21 +34,17 @@ class FormItemAlignHorizontal extends FormItemBase<
                     <div
                         className={
                             this.props.managedClasses
-                                .formItemAlignHorizontal_controlInputContainer
+                                .formItemTextAlign_controlInputContainer
                         }
                     >
                         {this.renderInput("left", 1)}
                         {this.renderInput("center", 2)}
                         {this.renderInput("right", 3)}
```

Comment:
What about justify? Technically text align should have 4 options.
![image](https://user-images.githubusercontent.com/37851214/53842799-5b6ab280-3f55-11e9-8b11-d14dacda6686.png)

Reply:
@eljefe223 I think this is a seperate issue currently and should likely stay seperate. This is additive, whereas we are prioritizing breaking changes currently. I think we can add this as part of a different PR